dp/create_id2title.py

```diff
@@ -14,95 +14,17 @@
 import argparse
 import gzip
 import logging
+from .dp_common import *
```

Comment:
this should be an absolute import in python3, and you should avoid using * whenever possible. Like `from dp.dp_common import something1,something2`